pkgs/development/r-modules/default.nix

```diff
@@ -216,508 +225,534 @@ let
   # `self` is `_self` with overridden packages;
   # packages in `_self` may depends on overridden packages.
   self = (defaultOverrides _self self) // overrides;
+
   _self = { inherit buildRPackage; } //
-          import ./bioc-packages.nix { inherit self; derive = deriveBioc; } //
+          import ./bioc-packages.nix            { inherit self; derive = deriveBioc; } //
           import ./bioc-annotation-packages.nix { inherit self; derive = deriveBiocAnn; } //
           import ./bioc-experiment-packages.nix { inherit self; derive = deriveBiocExp; } //
-          import ./cran-packages.nix { inherit self; derive = deriveCran; };
+          import ./bioc-workflows-packages.nix  { inherit self; derive = deriveBiocWrk; } //
+          import ./cran-packages.nix            { inherit self; derive = deriveCran; };
 
   # tweaks for the individual packages and "in self" follow
 
   packagesWithRDepends = {
-    FactoMineR = [ self.car ];
-    pander = [ self.codetools ];
+    # if R deps are missing for some reason, add them here.
+    tesseract = [ self.pdftools ];
+    TextForecast = [ self.Rpoppler ];
+    qmix = [ self.rstantools ]; # slow compile. Use linking and rstan?
+    cbq = [ self.rstantools ]; # slow compile. Use linking and rstan?
+    RBesT = [ self.rstantools ]; # slow compile. Use linking and rstan?
   };
 
-  packagesWithNativeBuildInputs = {
```

Comment:
Ditto. Please make such a stylistic change in a separate PR. As it is, it's hard hard to tell what you actually changed because the diff is so long.

Reply:
I am sure the diff could be improved and welcome suggestions. But the diff long mainly because it fixes hundreds of packages.

Reply:
Actually, the diff is unnecessarily long because you've replaced every explicit `pkgs` path with an implicit `with pkgs;` scope at the top of the attribute. There might be lots of build fixes included in the change you've submitted, but they aren't recognizable because of this noise.